outline.md

```diff
@@ -0,0 +1,17 @@
+## Motivation
+
+Introduce the document by describing the motivations for producing a consolidated set of software citation principles. This section is basically an overview of the different use cases/scenarios where it would be desirable to have a way to cite software.
+
```

Comment:
Yes, we had planned on using the existing use case list here—that tabular layout sounds good.